dotnet/samples/KernelSyntaxExamples/Example16_CustomLLM.cs

```diff
@@ -3,130 +3,109 @@
 using System;
 using System.Collections.Generic;
 using System.Runtime.CompilerServices;
-using System.Text;
-using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.SemanticKernel;
-using Microsoft.SemanticKernel.Connectors.OpenAI;
 using Microsoft.SemanticKernel.TextGeneration;
-using RepoUtils;
-
-#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
 
 /**
- * The following example shows how to plug into SK a custom text generation model.
+ * The following example shows how to plug a custom text generation model into SK.
+ *
+ * To do this, this example uses a text generation service stub (MyTextGenerationService) and
+ * no actual model.
  *
- * This might be useful in a few scenarios, for example:
+ * Using a a custom text generation model within SK can be useful in a few scenarios, for example:
  * - You are not using OpenAI or Azure OpenAI models
- * - You are using OpenAI/Azure OpenAI models but the models are behind a web service with a different API schema
```

Comment:
nit: I think it's still could be valid scenario, we can rephrase it if needed.

Reply:
Put back in